lib/config/rules/ecmascript-6.js

```diff
@@ -39,6 +39,8 @@ module.exports = {
   'prefer-arrow-callback': ['error', {allowNamedFunctions: true}],
   // Suggest using of const declaration for variables that are never modified after declared
   'prefer-const': 'warn',
+  // Require destructuring from arrays and/or objects <FEEDBACK>
+  'prefer-destructuring': ['error', {array: true, object: true}],
```

Comment:
I also feel this is better as only objects. I also kind of like the renaming option too